src/api/subscriptions.js

```diff
@@ -337,6 +337,13 @@ export class Subscriptions {
    * @return {?}
    */
   showBestAudienceAction() {}
+
+  /**
+   * Set the publisherProvidedId.
```

Comment:
nit: "Sets". Method descriptions should be third person sentences written as if they start with "This method...".

Reply:
Great, thanks for the example.